Cargo.toml

```
@@ -18,12 +18,12 @@ repository = "https://github.com/coreos/cargo-vendor-filterer"
 [dependencies]
 anyhow = "1.0"
 camino = "1.0"
-cargo-lock = "8"
-cargo_metadata = "0.11.3"
+cargo-lock = "10"
+cargo_metadata = "0.18.1"
```

```suggestion
cargo_metadata = "0.18"
```
I recommend not to specify the patch version, as it is the case for anyhow or camino crates.